### DISCUSSION TITLE
Order

### DISCUSSION THREAD

**Discussion by ma9004** [pytr-org/pytr#35] [General]
is it possible to place an order?

**Comment by Katzmann1983** [comment]
No(t yet)

**Reply by noname9312** [reply]
I also would love to create market buy orders. Is there any way this can be achieved?

**Reply by noname9312** [reply]
are you affiliated with this repo? https://github.com/ptrstn/pytr

it has market order but it hasnt been updated the last 4 years.

**Reply by Katzmann1983** [reply]
In theory the API has the feature to place an order. Not sure it is working or properly tested. Seems like somebody published an API in 2020 and then all repos grew from there.
Just try the order from the AP, e.g. market_order and share your experience.  :D
